src/screens/ShowDetailsScreen.tsx

```diff
@@ -1,13 +1,66 @@
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { getMovieProviders } from '../helpers/getMovieUtils';
+import { MovieProviders, MovieDetailsData } from '../types/tmdb';
+
 /**
  * Screen to show more details of a specific show
  * Rendered after user clicks on show card
  * 
  * @returns {JSX.Element}
  */
 export default function ShowDetailsScreen(): JSX.Element {
+	const [providers, setProviders] = useState<MovieProviders>();
+	const location = useLocation();
+	console.log(location);
+
+	useEffect(() => {
+		const handler = async () => {
+			const data = await getMovieProviders(location.state.details.id);
+			setProviders(data) as unknown as MovieProviders;
```

Comment:
We should be able to avoid this unknown conversion as this basically sidesteps type checking. I can look into this tonight as well, I know we had a similar issue elsewhere.